docs/tutorials/Hosting_on_Cloudflare_Workers.md

```diff
@@ -28,7 +28,7 @@ To start, we'll create the app through the [Discord Developer Dashboard](https:/
 
 ![IDs found in app settings](cloudflare-general-overview.png)
 
-- Now click on the `Bot` tab on the left sidebar, and create a bot! You can choose the same name as your app.
+- Now click on the **Bot** tab on the left sidebar.
 - Grab the `token` for your bot, and store it somewhere safe (I like to put these tokens in a password manager like [1password](https://1password.com/) or [lastpass](https://www.lastpass.com/)).
```

Comment:
Maybe recommend bitwarden instead of lastpass [given recent incidents](https://en.wikipedia.org/wiki/LastPass#Security_incidents)

Reply:
can probably update this to bitwarden or something, but will make a separate PR to keep this one focused on bot user stuff.